doc/SAI-Proposal-SAI-PTF.md

```diff
@@ -0,0 +1,132 @@
+# SAI PTF
+
+| Title       | SAI PTF         |
+| ----------- | --------------- |
+| Authors     | Intel           |
+| Status      | In review       |
+| Type        | Standards track |
+| Created     | 07/10/2021      |
+| SAI-Version | 1.9             |
+
+## Overview
+
+SAI PTF (Packet Testing Framework) is an autogenerated Python based dataplane testing framework. It is using thrift features to call C-based SAI functions in Python scripts. The main goal of the framework is to reduce the time spent on writing wrapper functions for SAI which could be generated based on SAI headers.
```

Comment:
auto-generated

Reply:
The main goal of the framework is to reduce the time spent on writing wrapper functions for SAI which can instead be generated based on SAI headers.

Reply:
*thrift wrapper functions